src/components/Container/Container.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Movie from '../Movie/Movie.js';
+
+const Container = props => {
+  const movieComponents = props.movies.map((movie, index) => (
+    <Movie
```

Comment:
We could destructure movie right out of props here to make things a bit more concise.

Reply:
seconded - you could even take this a step further and destructure the keys out of that movie, too